.circleci/config.yml

```diff
@@ -0,0 +1,171 @@
+version: 2
+workflows:
+  version: 2
+  tests:
+    jobs: &workflow_jobs
+      - node4:
```

Comment:
Don't bother supporting node.js 4, since it's EOL.  Please file a follow issue to drop it from this template, wherever you got it :)

Reply:
It copies it from some other random library, so when we remove node4 and node9 from all libraries, it will be automatically fixed :)

Note that due to lots of YAML references in this file removing node4 is not just removing node4. The easiest way to remove node4 manually without breaking anything is:
(1) remove all node6 sections
(2) rename node4 to node6 everywhere

Reply:
I'm gonna have your tool pull a sample circle config from `https://github.com/googleapis/nodejs-bigtable/blob/master/.circleci/config.yml`. It seems they have dropped node4 support (although node9 is still there)